src/Dfe.PlanTech.Infrastructure.Contentful/Helpers/ContentfulSetup.cs

```diff
@@ -36,13 +36,20 @@ public static IServiceCollection SetupContentfulClient(this IServiceCollection s
 
         services.AddTransient<IGetSubmissionStatusesQuery, GetSubmissionStatusesQuery>();
         services.AddTransient<ILogger<Category>, Logger<Category>>();
-        services.AddScoped<IContentfulClient, ContentfulClient>();
+        services.AddScoped<IContentfulClient>((services) =>
```

Comment:
Sorry this was left over from my debugging for the fix, you can remove lines 40-46 and just
```suggestion
        services.AddScoped<IContentfulClient, ContentfulClient>();
```

Reply:
[a7ee237](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/900/commits/a7ee23746b4af6efd015645300904b889a4e35c8)